MASH-FRET/.release_version.json

```diff
@@ -1,4 +1,4 @@
 {
 "tag" : "1.1.2",
-"prev_commit_hash" : "b8f2f71"
+"prev_commit_hash" : "d2484a0"
```

Comment:
I would propose a pushbutton that turns green and inactive once it has been pressed to indicate that recentering has taken place.

Reply:
Great suggestion, the checkbox was replaced by a regular pushbutton in MASH-code-6 that display a warning when all molecule positions are already recentered